cassovary-core/src/main/scala/com/twitter/cassovary/graph/Graph.scala

```diff
@@ -25,4 +27,6 @@ trait Graph {
   def getNodeById(id: Int): Option[Node]
 
   def existsNodeId(id: Int) = getNodeById(id).isDefined
+
+  val nodeLabels = new Labels[Int] //empty to start with
```

Comment:
Moving it to DirectedGraph. This allows Graph to be purer without any vals. Not much overhead to have an empty class included all the time for a collection like graph.